vars/mysql.yml

```diff
@@ -1 +1,2 @@
-mysql: "{{ vault_mysql }}"
\ No newline at end of file
+mysql: "{{ vault_mysql }}"
+mysql_user: "root"
```

Comment:
probably not needed if mysql_* ansible modules operate like the postgres ones

Reply:
apparently not? defaulting to netsoc